nomic/data_inference.py

```diff
@@ -77,7 +76,7 @@ class NomicProjectOptions(BaseModel):
     n_epochs: Optional[int] = None
     spread: Optional[float] = None
     local_neighborhood_size: Optional[int] = None
-    model: str = DEFAULT_PROJECTION_MODEL
```

Comment:
assuming we do this since pydantic in api already provides a default correct?

Reply:
Yep that's it! Last deployment covers this